### PR TITLE
impr(docker): manifest is now perfect

### PR DESCRIPTION
**Pull request by linuxbandit**
supersedes #200

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-            dockerfile: ./docker/Dockerfile
-        volumes:
-            - "./$PATH_OMS_FRONTEND/../:/usr/app/src"
+        image: aegee/oms-frontend:stable
+        #healthcheck?
+        cpu_count: 1
+        mem_limit: 900m
         labels:
-            - "traefik.frontend.rule=HostRegexp:{domain:[a-z0-9.]+};PathPrefix:/"
+            - "traefik.frontend.rule=Host:${SUBDOMAIN_FRONTEND}${BASE_URL};PathPrefix:/"
```
Comment (serge1peshcoff) [inline docker/docker-compose.yml]: This won't work under http://localhost right?

Comment (linuxbandit) [reply docker/docker-compose.yml]: it will, because the BASE_URL will be set to localhost

Comment (serge1peshcoff) [reply docker/docker-compose.yml]: I meant the SUBDOMAIN_FRONTEND part, it's "my" right?

File: docker/docker-compose.dev.yml [hunk 2]
```diff
@@ -7,4 +7,6 @@ services:
             dockerfile: ./docker/Dockerfile
         image: aegee/oms-frontend:latest
         volumes:
+            - /usr/app/src/node_modules
             - ./${PATH_OMS_FRONTEND}/../:/usr/app/src
+        command: npm install && npm run build && chmod 777 /usr/app/src && nginx
```
Comment (serge1peshcoff) [inline docker/docker-compose.dev.yml]: Idea: move this to .sh file

**Pull request merged**